semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -225,15 +211,17 @@ class SpanSemanticConvention(HasAttributes, BaseSemanticConvention):
 
     def __init__(self, group):
         super().__init__(group)
-        self._set_attributes(self.prefix, self.stability, group)
         self.span_kind = SpanKind.parse(group.get("span_kind"))
         if self.span_kind is None:
             position = group.lc.data["span_kind"]
             msg = "Invalid value for span_kind: {}".format(group.get("span_kind"))
             raise ValidationError.from_yaml_pos(position, msg)
 
 
-class EventSemanticConvention(HasAttributes, BaseSemanticConvention):
+print("MRO: ", SpanSemanticConvention.__mro__)
```

Comment:
```suggestion
```

Reply:
Done in a06432a (I accidentally pushed that to #64). Need to remove the newlines following that as well.